Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
I think we've seen pretty clear evidence in the past removing all basic usage nudges results in more issues being opened against Homebrew along the lines of x software doesn't work please help. As much as I agree with a lot of caveats being trimmed back I do think long-term you're increasing your own burden here. Not a discussion I expect to win, but FWIW.